R/xyplot.hydromad.R

```diff
@@ -138,7 +217,21 @@ qqmath.hydromad <-
     foo
   }
 
+
+#' @rdname xyplot.hydromad
+#' @export
 tsdiag.hydromad <- function(object, gof.lag, ...) {
   tsdiag.Arima <- getS3method("tsdiag", "Arima")
   tsdiag.Arima(object$uh, gof.lag = gof.lag, ...)
 }
+
+
+#' @rdname xyplot.hydromad
+#' @export
+plot.hydromad <-
```

Comment:
Where did this one move from?

Reply:
I think plot.hydromad was originally in the same file: [https://github.com/josephguillaume/hydromad/blob/388833b660b51add362f30f202c655e713714613/R/xyplot.hydromad.R](https://github.com/josephguillaume/hydromad/blob/388833b660b51add362f30f202c655e713714613/R/xyplot.hydromad.R).

Reply:
Looks like the reason it didn't show up is that the original location wasn't deleted - it's still at the top of this file, so this is a duplicate function definition.
I think it's ok to delete?

Reply:
Agreed. I've deleted it locally, it will appear with the next push.